cli/tools/lint/linter.rs

```diff
@@ -97,6 +99,15 @@ impl CliLinter {
     ext: Option<&str>,
   ) -> Result<(ParsedSource, Vec<LintDiagnostic>), AnyError> {
     let specifier = specifier_from_file_path(file_path)?;
```

Comment:
Files whose names end in `.min.js`, `.min.mjs`, or `.min.cjs` can be skipped.

Reply:
I'm not sure we should do it - this would bifurcate the logic and the "minified_file" will take care of analyzing if it makes sense to lint a file.